gallery/src/components/NxVulnerabilityIndicator/NxVulnerabilityIndicatorPage.tsx

```diff
@@ -0,0 +1,182 @@
+
+import NxVulnerabilityIndicatorBySeverityExample from './NxVulnerabilityIndicatorBySeverityExample';
+import NxVulnerabilityIndicatorByScoreExample from './NxVulnerabilityIndicatorByScoreExample';
+import NxVulnerabilityIndicatorListExample from './NxVulnerabilityIndicatorListExample';
+import NxVulnerabilityIndicatorTableExample from './NxVulnerabilityIndicatorTableExample';
+import NxVulnerabilityIndicatorWithTitleExample from './NxVulnerabilityIndicatorWithTitleExample';
+
+const nxVulnerabilityIndicatorBySeverityCode = require('./NxVulnerabilityIndicatorBySeverityExample?raw'),
+    nxVulnerabilityIndicatorByScoreCode = require('./NxVulnerabilityIndicatorByScoreExample?raw'),
+    nxVulnerabilityIndicatorListCode = require('./NxVulnerabilityIndicatorListExample?raw'),
+    nxVulnerabilityIndicatorTableCode = require('./NxVulnerabilityIndicatorTableExample?raw'),
+    nxVulnerabilityIndicatorWithTitleCode = require('./NxVulnerabilityIndicatorWithTitleExample?raw');
+
+const NxVulnerabilityIndicatorPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxCode>NxVulnerabilityIndicator</NxCode> is an inline element used
+        to indicate via color the IQ policy vulnerability rating of the information to follow that follows it.
+      </NxP>
+      <NxP>
+        There are two scales to choose from: vulnerability rating by category, and vulnerability
+        rating by score. When using this component, it is expected that just one of the props will be passed. If both
+        are passed, <NxCode>severityRating</NxCode> takes precedence.
+      </NxP>
+      <NxTable>
+        <NxTable.Head>
+          <NxTable.Row>
+            <NxTable.Cell>Prop</NxTable.Cell>
+            <NxTable.Cell>Type</NxTable.Cell>
+            <NxTable.Cell>Required</NxTable.Cell>
+            <NxTable.Cell>Details</NxTable.Cell>
+          </NxTable.Row>
+        </NxTable.Head>
+        <NxTable.Body>
+          <NxTable.Row>
+            <NxTable.Cell>severityRating</NxTable.Cell>
+            <NxTable.Cell>One of 'none', 'low', 'medium', 'high', or 'critical'</NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>A Vulnerability Severity Rating off of which to base the indicator color</NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell>score</NxTable.Cell>
+            <NxTable.Cell>0.0{'\u2014'}10.0 inclusive</NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>A Vulnerability (CVSS) Score off of which to base the indicator color</NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell>title</NxTable.Cell>
+            <NxTable.Cell>string</NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>
+              Specifies tooltip content, by default, it is set to the Vulnerability Severity Rating
```

Comment:
```suggestion
              Specifies tooltip content. By default, it is set to the Vulnerability Severity Rating
```
I think this also needs slightly more elaboration as the tooltip content isn't _just_ the rating name.

Reply:
Updated here: [07faf88](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/07faf8866585f8a06cc8e83275201a80e1b2d556)